.github/workflows/mend.yaml

```diff
@@ -0,0 +1,36 @@
+---
+name: Mend Monitor
+on:
+  push:
+    branches:
+      - 7.x
+      - main
```

Comment:
Should this PR target `main`? If we do target `7.x` does it get triggered twice since both 7.x and main branches will have the same "on push" trigger? I think for puppet-agent we were setting a different "on push branch" in each branch, since I don't know of a way to say "on push to *this* branch"

Reply:
I think that these behave more like an allow-list than a hook that will trigger per branch. I do think you are right that we should target `main` though.